Products/CMFPlone/controlpanel/browser/redirects.py

```diff
@@ -0,0 +1,280 @@
+# -*- coding: utf-8 -*-
+from AccessControl import getSecurityManager
+from cStringIO import StringIO
+from plone import api
```

Comment:
plone.api is not blessed into plone core packages 😞

Reply:
I have replaced the one line that used `plone.api`.